tox.ini

```diff
@@ -192,6 +197,7 @@ commands_pre =
   flask,django: pip install {toxinidir}/opentelemetry-auto-instrumentation
   flask: pip install {toxinidir}/ext/opentelemetry-ext-flask[test]
 
+  dbapi: pip install {toxinidir}/opentelemetry-auto-instrumentation
```

Comment:
Do you need the auto-instrumentation module for this test suite?

Reply:
not anymore; removed